ci-operator/step-registry/vsphere/provision/haproxy/vsphere-provision-haproxy-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: vsphere-provision-haproxy
+  from: upi-installer
+  cli: latest
+  grace_period: 10m
+  commands: vsphere-provision-haproxy-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: openshift-custom-mirror-registry
+    mount_path: /var/run/vault/vsphere
```

Comment:
I noticed the `/var/run/vault/vsphere` loaded _mirror registry_ content here, and _aws credential_ in `vsphere-provision-bastionhost` step, I suggest changing the mount point to something like `/var/run/vault/mirror-registry`, it's easy to understand, WDYT?

Reply:
agree, updated.